quicklogic/pp3/tests/features/install_test/counter/Makefile.symbiflow

```diff
@@ -0,0 +1,50 @@
+.PHONY:${BUILDDIR}
+
+current_dir := /home/mkurc/Repos/quicklogic-symbiflow-arch-defs/quicklogic/pp3/tests/features/install_test/counter
```

Comment:
This is pointing to local path. In this file there are multiple such instances.

Reply:
I have added this file by accident. It shouldn't be here. I'm going to remove it.